MyString.java

```diff
@@ -76,11 +134,15 @@ public static String randomStringOfLetters(int n) {
      * @param str1 - a string
      * @param str2 - a string
      * @return a string consisting of str1 minus all the characters of str2
-     */
+         */
+
     public static String remove(String str1, String str2) {
-       //// Replace the following statement with your code
-        return null;
+        for (char c : str2.toCharArray()) {
+            str1 = str1.replaceFirst(String.valueOf(c), "");
+        }
+        return str1;

```

Comment:
this is literally solving the solution. this is a violation of HW guidelines. 
-2 points